modules/apps/product-navigation/product-navigation-applications-menu-web/src/main/resources/META-INF/resources/js/ApplicationsMenu.js

```diff
@@ -346,6 +351,28 @@ const ApplicationsMenu = ({
 		onClose: () => setVisible(false),
 	});
 
+	useEventListener(
+		'keydown',
+		(event) => {
+			const isCMDPressed = event.metaKey || event.ctrlKey;
+
+			if (
+				isCMDPressed &&
+				event.shiftKey &&
+				event.key.toLowerCase() === 'm'
+			) {
+				event.preventDefault();
+
+				if (!visible) {
+					fetchCategories();
+				}
+				setVisible(!visible);
```

Comment:
Add an break line here, just so everyone's happy :)

Reply:
I'll let the code breathe a little here

Reply:
f4f550b